compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/skiko/RecordDrawRectRenderDecorator.skiko.kt

```diff
@@ -39,6 +39,7 @@ internal class RecordDrawRectRenderDecorator(
             }
         }
 
+    // TODO: nobody calls it
```

Comment:
Missing TODO?

Reply:
cc @MatkovIvan , I will remove this TODO. Please open a ticket.

Reply:
https://youtrack.jetbrains.com/issue/CMP-6590